go/packages/slslambda/slslambda.go

```diff
@@ -47,7 +48,16 @@ func CaptureError(ctx context.Context, err error) {
 		debugLog("capture error:", ctxErr)
 		return
 	}
-	span.captureError(err)
+	span.captureError(err, nil)
+}
+
+func CaptureErrorWithOptions(ctx context.Context, err error, options EventOptions) {
```

Comment:
Is it typical to have two functions where one says `WithOptions` in go?

Reply:
I see it pretty commonly, go doesn't support method overloading or default parameters, so if we wanted to add options to the existing `CaptureError` method it would be a breaking change. This way we don't break existing calls to that method.